cross/libvpx/Makefile

```diff
@@ -46,6 +46,60 @@ ifeq ($(findstring $(ARCH),$(PPC_ARCHES)),$(ARCH))
 CONFIGURE_ARGS += --target=generic-gnu
 endif
 
+# Removal of GCC additional warning flags required at least for ppc853x-5.2 qoriq-5.2
```

Comment:
See my other comment on the patch file first.

This can probably be cleaned up a fair bit: if all PowerPC-based arches need this change (and it looks like it), than just add the MAKE_ARGS` to https://github.com/SynoCommunity/spksrc/blob/master/cross/libvpx/Makefile#L45.